src/exporter/DaemonMetricCollector.cc

```diff
@@ -422,6 +429,7 @@ void DaemonMetricCollector::update_sockets() {
       std::string daemon_name =
           daemon_socket_name.substr(0, daemon_socket_name.size() - 5);
       if (clients.find(daemon_name) == clients.end() &&
+          !(daemon_name.find("client") != std::string::npos) &&
```

Comment:
If this list explicitly excludes `client`s, shouldn't it be better called `daemons`?

Reply:
This is the list of asok files used in the host. The name used in the socket file depend of the "process" using the socket file. 
And you pointed an error i did!!!.
 
The RGW gateways use asok file names as:
ceph-client.rgw.rgwfoo.chatest-node-00.yqaoen.2.93915532417312.asok

So is not correct to exclude asok files "with the word "client".

Reply:
If it starts with `client` string you can use [C++20 `starts_with()`](https://en.cppreference.com/w/cpp/string/basic_string/starts_with).

Reply:
The error is to not take into account asok files with the "client" word inside.